stock_warehouse_relationship/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2019 Akretion
+# Copyright 2022 Foodles (http://www.foodles.co).
+# @author Pierre Verkest <pierreverkest84@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Warehouse relationship",
+    "summary": "Technical module to add warehouse_id field on various stock.* models",
+    "version": "16.0.1.0.0",
+    "category": "Warehouse Management",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "author": "Akretion, Pierre Verkest, Odoo Community Association (OCA)",
+    "maintainers": ["petrus-v"],
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": [
+        "stock",
```

Comment:
just notice that this module is based on  `stock_location_warehouse`
```suggestion
        "stock",
        "stock_location_warehouse",
```

Reply:
I don't think  `stock_location_warehouse` is necessary in v16.
`warehouse_id` field on `stock.location` is present in `stock` module.

Reply:
ok great I haven't deep into it on v16!